workspaces/ui-v2/src/pages/docs/EndpointRootPage.tsx

```diff
@@ -347,6 +433,7 @@ const useStyles = makeStyles((theme) => ({
   bodyContainer: {
     marginTop: theme.spacing(6),
     width: '100%',
+    height: '100%',
```

Comment:
Some more slightly suss container css properties here. If a `min-height: 100vh` achieves the same thing, that would be preferable!